### PR TITLE
appveyor: support master and latest release packages

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -36,8 +40,15 @@ install:
   - rmdir /S /Q storage\mroonga\
   - ps: Get-Content ..\mroonga\version | ForEach-Object { $env:MROONGA_VERSION = $_ }
   - move ..\mroonga storage\mroonga
-  - git clone --quiet --depth 1 --recursive https://github.com/groonga/groonga.git ..\groonga
-  - ps: Get-Content ..\groonga\base_version | ForEach-Object { $env:GROONGA_VERSION = $_ }
+  - if "%USE_MASTER%" == "yes" (
+       git clone --quiet --depth 1 --recursive https://github.com/groonga/groonga.git ..\groonga
+    )
+  - if not "%USE_MASTER%" == "yes" (
```
Comment (komainu8) [inline appveyor.yml]: We should use "else".

File: appveyor.yml [hunk 2]
```diff
@@ -46,7 +57,17 @@ install:
   - cd ..\..\mariadb-%MARIADB_VERSION%
   - mkdir storage\mroonga\vendor
   - move ..\groonga storage\mroonga\vendor\groonga
-  - git clone --quiet --depth 1 https://github.com/groonga/groonga-normalizer-mysql.git storage\mroonga\vendor\groonga\vendor\plugins\groonga-normalizer-mysql
+  - if "%USE_MASTER%" == "yes" (
+      git clone --quiet --depth 1
+        https://github.com/groonga/groonga-normalizer-mysql.git
+        storage\mroonga\vendor\groonga\vendor\plugins\groonga-normalizer-mysql
+    )
+  - if "%USE_MASTER%" == "" (
```
Comment (komainu8) [inline appveyor.yml]: Ditto.

File: appveyor.yml [hunk 3]
```diff
@@ -36,8 +40,15 @@ install:
   - rmdir /S /Q storage\mroonga\
   - ps: Get-Content ..\mroonga\version | ForEach-Object { $env:MROONGA_VERSION = $_ }
   - move ..\mroonga storage\mroonga
-  - git clone --quiet --depth 1 --recursive https://github.com/groonga/groonga.git ..\groonga
-  - ps: Get-Content ..\groonga\base_version | ForEach-Object { $env:GROONGA_VERSION = $_ }
+  - if "%USE_MASTER%" == "yes" (
+       git clone --quiet --depth 1 --recursive https://github.com/groonga/groonga.git ..\groonga
```
Comment (komainu8) [inline appveyor.yml]: Is processing of getting Groonga base version needless?

File: appveyor.yml (hunk 1 above)
Comment (kenhys) [reply appveyor.yml]: I didn't notice that `else` works as expected. I've fixed.

File: appveyor.yml (hunk 2 above)
Comment (kenhys) [reply appveyor.yml]: Fixed.

File: appveyor.yml (hunk 3 above)
Comment (kenhys) [reply appveyor.yml]: Sure, now, there is no need to use $env:GROONGA_VERSION.

**Comment by kenhys** [comment]
Thank you for your review, I've fixed them.
@komainu8 could you review it again, please?

**Comment by kenhys** [comment]
@kou could you review this PR, please?

**Pull request merged**